src/main/java/frc/robot/Constants.java

```diff
@@ -96,6 +99,29 @@ public final static class DriverConstants {
         public final static double speedMultiplier = 3;
         public final static double angleMultiplier = 3;
     }
+    
+    public final static class LEDConstants {
+        public final static int BLINKIN_CHANNEL = 0;
+    }
+
+    public enum LEDColors {
```

Comment:
On reflection, calling this LEDColor would make more sense, as whenever you refer to the enum you're only doing so in reference to a single color